test/config/mocks/reset-dynamic-directive-injections.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Angular providers do not reset between each test, only service instance do.
+ */
+angular
+  .module('linagora.esn.tests.reset-dynamic-directive-injections', [
+    'op.dynamicDirective'
```

Comment:
Question: Since we do not support dynamic directive, do we need this?

Reply:
No, we don't need this.